src/Microsoft.AspNetCore.SystemWebAdapters.SessionState/RemoteSession/RemoteAppSessionStateManager.cs

```diff
@@ -64,7 +63,7 @@ public async Task<ISessionState> CreateAsync(HttpContextCore context, SessionAtt
         try
         {
             // Get or create session data
-            var response = await GetSessionDataAsync(sessionId, metadata.IsReadOnly, context, cts.Token);
+            var response = await GetSessionDataAsync(sessionId, metadata.IsReadOnly, context, context.RequestAborted);
```

Comment:
oh I like how this simplifies the cancellation logic - no more custom sources :)